_includes/head.html

```diff
@@ -14,6 +14,16 @@
     {%- assign c = b | split: '>' | last -%}
     {%- assign page_title = c | strip_html | strip | truncatewords: 10 -%}
 {%- endif -%}
+{%- if page.description == nil and page.datafile != nil and page.datafile != '' -%}
+    {%- assign yaml_data = site.data[page.datafolder][page.datafile] -%}
+    {%- if yaml_data.long and yaml_data.long != '' -%}
+        {%- assign page_description = page.title | append: ': ' | append: yaml_data.long | strip_html | strip | truncatewords: 30 -%}
+    {%- elsif yaml_data.short and yaml_data.short != '' -%}
```

Comment:
FWIW, we can swap order of priority here if we want; the short description may be "too short", but the "long" description can contain HTML, tables, etc, so for some pages may contain some irrelevant info.

That's something we could tweak later of course; we can also manually create descriptions in the markdown-stubs (or update the short command description upstream in docker/cli)